app/models/assignment.rb

```diff
@@ -12,6 +12,7 @@ class Assignment < ApplicationRecord
            primary_key: :submission_id,
            dependent: :destroy
 
+  belongs_to :organization, default: -> { Organization.current }
```

Comment:
Will the organization be updated if it it is resubmitted from another organization?

Reply:
This is changed now and it doesn't depend on default values anymore.